silkrpc/ethbackend/backend.hpp

```diff
@@ -1,175 +1,40 @@
-    ClientVersionClient,
-    ::remote::ETHBACKEND::StubInterface,
-    ::remote::ClientVersionRequest,
-    ::remote::ClientVersionReply
->;
-
-using NetPeerCountAwaitable = unary_awaitable<
-    asio::io_context::executor_type,
-    NetPeerCountClient,
-    ::remote::ETHBACKEND::StubInterface,
-    ::remote::NetPeerCountRequest,
-    ::remote::NetPeerCountReply
->;
-
-using EngineGetPayloadV1Awaitable = unary_awaitable<
-    asio::io_context::executor_type,
-    EngineGetPayloadV1Client,
-    ::remote::ETHBACKEND::StubInterface,
-    ::remote::EngineGetPayloadRequest,
-    ::types::ExecutionPayload
->;
-
-class BackEnd final {
-public:
-    explicit BackEnd(asio::io_context& context, std::shared_ptr<grpc::Channel> channel, grpc::CompletionQueue* queue)
-    : BackEnd(context.get_executor(), ::remote::ETHBACKEND::NewStub(channel), queue) {}
-
-    explicit BackEnd(asio::io_context::executor_type executor, std::unique_ptr<::remote::ETHBACKEND::StubInterface> stub, grpc::CompletionQueue* queue)
-    : executor_(executor), stub_(std::move(stub)), queue_(queue) {
-        SILKRPC_TRACE << "BackEnd::ctor " << this << "\n";
-    }
-
-    ~BackEnd() {
-        SILKRPC_TRACE << "BackEnd::dtor " << this << "\n";
-    }
-
-    asio::awaitable<evmc::address> etherbase();
-    asio::awaitable<uint64_t> protocol_version();
-    asio::awaitable<uint64_t> net_version();
-    asio::awaitable<std::string> client_version();
-    asio::awaitable<uint64_t> net_peer_count();
-    asio::awaitable<ExecutionPayload> engine_get_payload_v1(uint64_t payload_id);
-
-    // just for testing
-    asio::awaitable<::types::ExecutionPayload> execution_payload_to_proto(ExecutionPayload payload);
-
-private:
-    evmc::address address_from_H160(const types::H160& h160);
-    silkworm::Bytes bytes_from_H128(const types::H128& h128);
-    types::H128* H128_from_bytes(const uint8_t* bytes);
-    types::H160* H160_from_address(const evmc::address& address);
-    types::H256* H256_from_bytes(const uint8_t* bytes);
-    silkworm::Bytes bytes_from_H256(const types::H256& h256);
-    intx::uint256 uint256_from_H256(const types::H256& h256);
-    types::H256* H256_from_uint256(const intx::uint256& n);
-    evmc::bytes32 bytes32_from_H256(const types::H256& h256);
-    types::H512* H512_from_bytes(const uint8_t* bytes);
-    silkworm::Bytes bytes_from_H512(types::H512& h512);
-    types::H1024* H1024_from_bytes(const uint8_t* bytes);
-    silkworm::Bytes bytes_from_H1024(types::H1024& h1024);
-    types::H2048* H2048_from_bytes(const uint8_t* bytes);
-    silkworm::Bytes bytes_from_H2048(types::H2048& h2048);
-
-    ExecutionPayload decode_execution_payload(const types::ExecutionPayload& execution_payload_grpc);
-    types::ExecutionPayload encode_execution_payload(const ExecutionPayload& execution_payload);
-
-    asio::io_context::executor_type executor_;
-    std::unique_ptr<::remote::ETHBACKEND::StubInterface> stub_;
-    grpc::CompletionQueue* queue_;
+class BackEnd {
+    public:
```

Comment:
Indentation at line 30 (just because I have to put at least one comment :smiley:)